cmd/lnmuxd/run.go

```diff
@@ -249,18 +235,47 @@ func run(ctx context.Context, cfg *Config) error {
 		return err
 	}
 
-	group, ctx := errgroup.WithContext(ctx)
-
 	// Run multiplexer.
-	group.Go(func() error {
-		err := mux.Run(ctx)
+	var muxes []*lnmux.Mux
+	for _, lnd := range lnds {
+		mux, err := lnmux.New(
```

Comment:
@Torakushi, I made a change to first instantiate all muxes and only then spin up the goroutines. Otherwise we may have lingering goroutines if there is an error.

Reply:
Nice catch! Remind me @91 :)